search-attributes/src/workflows.ts

```diff
@@ -1,3 +1,14 @@
+import { upsertSearchAttributes, workflowInfo } from '@temporalio/workflow';
+
+// @@@SNIPSTART typescript-search-attributes-workflow
 export async function example(): Promise<string> {
-  return 'hello workflow';
+  const customInt = (workflowInfo().searchAttributes?.CustomIntField[0] as number) || 0;
+  upsertSearchAttributes({
```

Comment:
Show how to delete (set to `[]`) and show that you can read modified search attributes